### ISSUE TITLE
increased/decreased ganglioside level

### ISSUE THREAD

**Issue by slaulederkind** [mgijax/mammalian-phenotype-ontology#3887]
>If this is a request for a new term, please provide us with the following information.

>1. Preferred term label (e.g., asplenia)

1. increased ganglioside level
2. decreased ganglioside level

>2. Synonyms (e.g., absent spleen)

1. amplified ganglioside level
1. accumulation of ganglioside
2. diminished ganglioside level
2. reduced ganglioside level

>3. Textual definition (the definition should be understandable even for non-specialists. Include a PubMed ID or URL(s) to refer to any relevant article(s) that provides information about the suggested term).

1. Increase in the concentration of the molecule composed of a glycosphingolipid (ceramide and oligosaccharide) with one or more sialic acids linked on the sugar chain
2. Decrease in the concentration of the molecule composed of a glycosphingolipid (ceramide and oligosaccharide) with one or more sialic acids linked on the sugar chain

>4. Any additional notes to clarify the meaning or use of the term. Please provide a PubMed ID or URL(s) for the source of this information.

PMID:34928329

>5. Suggested parent term (Please look in the hierarchy of the MP using the [MP Browser](http://www.informatics.jax.org/vocab/mp_ontology) or other browsers such as [OLS](http://www.ebi.ac.uk/ols/ontologies/mp) and find the place in the MP where the new term best fits)

abnormal ganglioside level (MP:0012231)

>6. If you would like a nanoattribution, please indicate your ORCID id or provide a label such as MGI:sbello (organization, first initial-last name)

ORCID:0000-0001-5356-4174
